packages/jumpstarter/jumpstarter/client/lease.py

```diff
@@ -40,8 +40,9 @@ def __post_init__(self):
     async def _create(self):
         duration = duration_pb2.Duration()
         duration.FromSeconds(self.timeout)
+        duration_str = f"{duration.seconds}s"
```

Comment:
without this we had an extra line jump at the end.

Reply:
What about
```suggestion
       duration_str = str(duration.ToTimedelta())
```
It shall be able to show the time in HH:MM:SS, which is far easier to read for large durations.

Reply:
I will try in a sec